python/ray/worker.py

```diff
@@ -2033,6 +2127,13 @@ def fetch_and_execute_function_to_run(key, worker=global_worker):
             data={"name": name})
 
 
+def register_existing_class(worker=global_worker):
+    export_keys = worker.redis_client.lrange("Exports", 0, -1)
```

Comment:
Does it query all existing `Exports`? 
I guess it might hurt perf this way.
Is it possible to only query `RegisterType` or only query `RegisterType` with a given driver id?

Reply:
It is a list in redis, we can't query subset of it.

Reply:
I think we can move the Register type out of the Exports, it will make the code more readable and improve the perf.

Reply:
Agree.